maven-repository-metadata/src/main/mdo/metadata.mdo

```diff
@@ -141,6 +141,14 @@ under the License.
                 }
             }
 
+            for ( SnapshotVersion version : versioning.getSnapshotVersions() )
+            {
+                if ( !v.getSnapshotVersions().contains( version ) )
```

Comment:
I guess this needs more refinement as IMHO only extension, version, and classifier act as id. The `updated` value should always be taken from the newer version I guess....